src/main/java/com/mg105/utils/ItemConstants.java

```diff
@@ -1,17 +1,23 @@
 package com.mg105.utils;
 
 
+import com.mg105.entities.items.HealthPotion;
+
 public class ItemConstants {
 
     public static final String UPGRADE_TOKEN_NAME = "Upgrade Token";
-    public static final String UPGRADE_TOKEN_DESCRIPTION = "One use increases a " + "specific stat on a specific character by one.";
+    public static final String UPGRADE_TOKEN_DESCRIPTION = "One use increases a " +
+        String.valueOf(HealthPotion.getHealingPoints()) + "specific stat on a specific character by one.";
```

Comment:
i think this is a typo? because it's an upgrade token so it wouldn't have to do with HealthPotion

Reply:
Oh, also this goes back to the question of if the token should upgrade all or one stat